dbaas/dbaas/settings.py

```diff
@@ -180,6 +180,7 @@ def LOCAL_FILES(path):
     'account',
     'admin',
     'adminplus',
+    'djcelery_dbaas',
```

Comment:
Essa app existe?

Reply:
yes and it is there to override the djcelery app